packages/suite/src/views/wallet/send/components/Options/components/BitcoinOptions/index.tsx

```diff
@@ -55,6 +70,25 @@ const BitcoinOptions = () => {
     const rbfEnabled = options.includes('bitcoinRBF');
     const broadcastEnabled = options.includes('broadcast');
 
+    const { openNode, openGuide } = useActions({
+        openGuide: guideActions.open,
+        openNode: guideActions.openNode,
+    });
+
+    const tooltipOpensGuide = (id: string) => {
+        const node: Node = {
+            type: 'page',
+            id: '',
+            locales: [''],
+            title: {
+                '': 'string',
+            },
+        };
```

Comment:
You can use `getNodeById` in `@suite-utils/guide`.

I would suggest creating a helper function `openGuideNodeById`.

```
const openGuideNodeById = (id: string) => {
    const node = getNodeById(id);
    if (!node) return; // + log error
    openNode(node);
    openGuide();
}
```